tubular/edx_api.py

```diff
@@ -496,3 +496,17 @@ def retire_learner(self, learner):
         except HttpDoesNotExistException:
             LOG.info("No license manager data found for user")
             return True
+
+class CommerceCoordinatorApi(BaseApiClient):
+    """
+    Commerce-Coordinator API client.
+    """
+    @_retry_lms_api()
+    def retire_learner(self, learner):
+        """
+        Performs the learner retirement step for Commerce-Coordinator.
+        Passes the learner's LMS User Id instead of username.
+        """
+        data = {'edx_lms_user_id': learner['user']['id']}
+        api_url = self.get_api_url('lms/user_retirement')
```

Comment:
Gosh, I always found this so confusing. So the endpoint/path is in commerce coordinator and the url path to the retirement endpoint is {blah_host}/lms/user_retirement?

I think having "lms" as part of it was throwing me off, I thought it was referring to the monolith.

Reply:
hmm. verifying this. 😆